### PR TITLE
Cloud 3017 restore doesn t work on azure

### PR DESCRIPTION
**Pull request by maxres-fr**

### CODE REVIEW

**Review by jrcast — APPROVED** [review]
LGTM

File: pkg/secretsmanager/secretsmanager.go [hunk 1]
```diff
@@ -401,6 +410,9 @@ func (sm *secretManagerAWS) CloseClient() {}
 // EnsureSecret saves secret to AWS secret manager
 func (sm *secretManagerAWS) EnsureSecret(ctx context.Context, secretName string, value []byte) error {
 	// get secret ID
+	if binary.Size(value) > awssecretsManagerMaxBytes {
+		return errors.WithStack(fmt.Errorf("unable to write %s to azure vault secret exceeds 65kb", secretName))
```
Comment (shankar-forgerock) [inline pkg/secretsmanager/secretsmanager.go:414]: Suggestion: May be better to keep the error messages for Secret Manager and key vault secrets distinct from each other.
s/azure vault secret/secret manager secret/

**Review by shankar-forgerock — APPROVED** [review]

**Review by dgoldssfo — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -208,6 +208,9 @@ data:
   # AZURE_CLIENT_SECRET: # OPTIONAL: Update if using Azure Key Vault
 ```
 
+
+**Note: The maximum secret size supported by Azure is 25Kb** For more information, see [Azure documentation](https://docs.microsoft.com/en-us/azure/key-vault/secrets/about-secrets).
```
Comment (wstrange) [inline README.md:212]: 🤦

**Review by wstrange — APPROVED** [review]

**Review by lee-baines — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (maxres-fr) [reply README.md:212]: The actual bug was encoding not size.

**Pull request merged**